frontend/packages/dev-console/src/components/import/import-validation-utils.ts

```diff
@@ -6,7 +6,7 @@ import { isInteger } from '../../utils/yup-validation-util';
 const urlRegex = /^(((ssh|git|https?):\/\/[\w]+)|(git@[\w]+.[\w]+:))([\w\-._~/?#[\]!$&'()*+,;=])+$/;
 const hostnameRegex = /^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$/;
 const pathRegex = /^\/.*$/;
-const relativePathRegex = /^[a-zA-Z0-9]([a-zA-Z0-9-]*[a-zA-Z0-9])?(\/.[a-zA-Z0-9]([a-zA-Z0-9-]*[a-zA-Z0-9])?)*$/;
+const relativePathRegex = /^[a-zA-Z0-9]([\w-.]*[a-zA-Z0-9])?(\/.[a-zA-Z0-9]([\w-.]*[a-zA-Z0-9])?)*$/;
```

Comment:
i know we don't have comments for other regex but it'll good to have comment as well, WDYT?